.vscode/extensions.json

```diff
@@ -2,6 +2,16 @@
   "recommendations": [
     "editorconfig.editorconfig",
     "redhat.vscode-yaml",
-    "biomejs.biome"
+    "biomejs.biome",
```

Comment:
<img width="592" height="190" alt="Screenshot 2025-07-15 at 3 53 39 PM" src="https://github.com/user-attachments/assets/a9db4b81-b73b-4711-a7de-92c053911e20" />

my one recommendation

Reply:
I am going to go with @ArrayKnight 's opinions here. While I like that package to generate colors, I also agree that moving towards the standard TK makes sense, and makes this less of a priority for most devs/designers.